include/z64.h

```diff
@@ -184,10 +184,22 @@ typedef struct {
     /* 0x0104 */ Vec3f  curDistortionOrientation;
     /* 0x0110 */ Vec3f  curDistortionScale;
     /* 0x011C */ u16    normal; // used to normalize the projection matrix
-    /* 0x0120 */ s32    flags;
+    /* 0x0120 */ s32    dirty;
     /* 0x0124 */ s32    unk_124;
 } View; // size = 0x128
 
+#define VIEW_VIEWING 0x1
+#define VIEW_VIEWPORT 0x2
+#define VIEW_PROJECTION_PERSPECTIVE 0x4
+#define VIEW_PROJECTION_ORTHOGRAPHIC 0x8
+#define VIEW_ALL 0xF
+
+#define VIEW_FORCE_VIEWING 0x10
+#define VIEW_FORCE_VIEWPORT 0x20
+#define VIEW_FORCE_PROJECTION_PERSPECTIVE 0x40
+#define VIEW_FORCE_PROJECTION_ORTHOGRAPHIC 0x80
```

Comment:
Since the code shifts these right by 4 to get the un-forced flags, there's an intrinsic relationship between their values and changing one define in one set but not the other can cause unintended side-effects. I suggest defining these as the other flag define << 4 to make this totally clear.

Reply:
I did try that, but to be honest I kind of prefer this. Couple of reasons:
* I find this much easier to read, and it becomes a bit of a "mess of capital letters" the other way, so much so that to my eye it obscures the relationship it's trying to clarify.
* It's impossible to define any more flags without changing the whole shifting logic anyway.

Reply:
Done 35f4fae87